ably/options_test.go

```diff
@@ -2,11 +2,195 @@ package ably_test
 
 import (
 	"net/url"
+	"reflect"
 	"testing"
 
 	"github.com/ably/ably-go/ably"
+	"github.com/stretchr/testify/assert"
 )
 
+func TestDefaultFallbacks_RSC15h(t *testing.T) {
+	t.Parallel()
+	t.Run("with env should return environment fallback hosts", func(ts *testing.T) {
+		expectedFallBackHosts := []string{
+			"a.ably-realtime.com",
+			"b.ably-realtime.com",
+			"c.ably-realtime.com",
+			"d.ably-realtime.com",
+			"e.ably-realtime.com",
+		}
+		hosts := ably.DefaultFallbackHosts()
+		assert.True(ts, reflect.DeepEqual(expectedFallBackHosts, hosts), " %s should match %s", hosts, expectedFallBackHosts)
+	})
+}
+
+func TestEnvFallbackHosts_RSC15i(t *testing.T) {
+	t.Parallel()
+	t.Run("with env should return environment fallback hosts", func(ts *testing.T) {
+		expectedFallBackHosts := []string{
+			"sandbox-a-fallback.ably-realtime.com",
+			"sandbox-b-fallback.ably-realtime.com",
+			"sandbox-c-fallback.ably-realtime.com",
+			"sandbox-d-fallback.ably-realtime.com",
+			"sandbox-e-fallback.ably-realtime.com",
+		}
+		hosts := ably.GetEnvFallbackHosts("sandbox")
+		assert.True(ts, reflect.DeepEqual(expectedFallBackHosts, hosts), " %s should match %s", hosts, expectedFallBackHosts)
```

Comment:
Suggestion: If we're using testify after all, we might as well make full use of it:

```suggestion
		assert.Equal(ts, expectedFallBackHosts, hosts)
```

This will emit an appropriate error message, with diff and all.